Resources/Prototypes/Roles/Antags/revolutionary.yml

```
@@ -18,7 +18,8 @@
   setPreference: true
   objective: roles-antag-rev-head-objective
   requirements:
-    - !type:OverallPlaytimeRequirement
+   - !type:DepartmentTimeRequirement # was  - !type:OverallPlaytimeRequirement, Goob change
```

```suggestion
  - !type:DepartmentTimeRequirement # was  - !type:OverallPlaytimeRequirement, Goob change
```
:face_holding_back_tears: